backend/create_superuser.py

```
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
```

Ruff (0.14.3)

Shebang is present but file is not executable

(EXE001)